api/proto/disperser/disperser.proto

```diff
@@ -25,6 +27,32 @@ service Disperser {
 
 // Requests and Responses
 
+// Authenicated Message Types
+
+message AuthenticatedRequest {
+    oneof payload {
+        DisperseBlobRequest disperse_request = 1;
+        ChallengeReply challenge_reply = 2;
```

Comment:
AuthenticationRequest/AuthenticationReply (or AuthRequest/AuthReply)? The "challenge" is more like the approach we take here.

Reply:
I renamed the paramters. Let me know if this looks better!